### DISCUSSION TITLE
Error in quadtree example on a Winblows system

### DISCUSSION THREAD

**Discussion by laat0003** [pypest/pypestutils#25] [General]
Hi,

I'm applying the fancy pilot points and grid-scale parameterisation (with a few GPLANES for fun) to a project I'm currently working on. So far its working a treat, my thanks to the devs.

That said, I believe that there may be an error in the quadtree example that is specific to anyone using Windows. In the example, where the grid-scale parameters are added to the  PstFrom object, the base filename string includes a colon. This has repercussions for writing the pilot-point csv's, which end up as empty files with no extension on a Windows system. The code still executes fine and despite the files appearing as empty, the parameters are added to the PstFrom object.  

Thanks again,
Tariq

![image](https://github.com/pypest/pypestutils/assets/49386249/4972dc5c-810c-4da7-adf4-a175b91c51c3)

**Comment by jtwhite79** [comment]
@laat0003 wanna take a shot at fixing this issue and opening a PR?

**Reply by laat0003** [reply]
Ha. @jtwhite79 surely the fact that I didn't think of that first should tell you how much of a dumbass I am :)
I'll try and get one sorted tonight - cheers for the prompt.
